README.md

```diff
@@ -1,334 +1 @@
-       --set global.jfrog.observability.metrics.token=<JPD_ADMIN_TOKEN> \
-       --set global.jfrog.observability.branch=master
-```
-
-## Fluentd Configuration for Elastic
-
-All the dynamic configuration for fluentd is picked from environment variable for xray and artifactory.
-
-### Fluentd configs:
-
-- **Artifactory** - https://raw.githubusercontent.com/jfrog/log-analytics-elastic/master/fluent.conf.rt
-- **Xray** - https://raw.githubusercontent.com/jfrog/log-analytics-elastic/master/fluent.conf.xray
-
-## Dashboards
-
-### Artifactory dashboard
-
-- **Application** - This dashboard tracks Log Volume(information about different log sources) and Artifactory Errors over time(bursts of application errors that may otherwise go undetected)
-- **Audit** - This dashboard tracks audit logs help you determine who is accessing your Artifactory instance and from where. These can help you track potentially malicious requests or processes (such as CI jobs) using expired credentials.
-- **Docker** - This dashboard tracks dockerhub pull request changes (anonymous gets only 100 requests per 6 hours where as free account gets 200 requests per 6 hours)
-- **Requests** - This dashboard tracks HTTP response codes, Top 10 IP addresses for uploads and downloads
-- **Xray Logs** - This dashboard tracks Log Volume, Xry Log Errors over time, HTTP 500 errors and Xray HTTP Response codes
-- **Xray Violations** - This dashboard gives an aggregated snapshot of all the violations, watch policies, and the total number of infected artifacts and components within their environment. This information is also organized by watch, by policy, by type and by severity to provide deeper segmentation and analysis.
-
-## Elastic Demo
-
-# Demo Requirements
-
-- Kubernetes Cluster
-- Artifactory and/or Xray installed via [JFrog Helm Charts](https://github.com/jfrog/charts)
-- Helm 3
-
-### Installing Elasticsearch and Kibana on K8s
-
-Elasticsearch kibana setup can be done using the following files or using manual configuration
-
-Install ECK operator following https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html
-
-- [Elasticserach Cluster](https://github.com/jfrog/log-analytics-elastic/blob/master/elastic/k8s-operator/elasticsearch_cluster.yaml) - Elasticsearch Cluster
-- [Kibana](https://github.com/jfrog/log-analytics-elastic/blob/master/elastic/k8s-operator/kibana.yaml) - Kibana
-
-### Setup
-
-To run this integration start by creating elasticsearch and kibana
-
-```
-kubectl create -f elastic/k8s-operator/elasticsearch_cluster.yaml
-kubectl create -f elastic/k8s-operator/kibana.yaml
-```
-
-## References
-
-- [Elasticsearch](https://www.elastic.co/) - Elastic search log data platform
-- [Kibana](https://www.elastic.co/kibana) - Elastic search visualization layer
+# Important: The integration will reach End of Support on September 30, 2024. After that date, this repository will no longer receive updates from JFrog and will no longer be supported by JFrog
```

Comment:
Shouldn't this be an *addition* to the existing README content? i.e. not a complete replacement.

Reply:
Agree!